Samples/Flix/src/main/java/dev/testify/samples/flix/domain/model/MovieDomainModel.kt

```diff
@@ -0,0 +1,17 @@
+package dev.testify.samples.flix.domain.model
+
+import kotlinx.datetime.Instant
+
+data class MovieDomainModel(
```

Comment:
e.g. Following off above, `Movie` is just much nicer to me than `MovieDomainModel`